src/main/webapp/components/result-export/result-export.js

```diff
@@ -49,8 +50,12 @@ const useExportSetMutation = () => {
 }
 
 const Container = props => {
+  const [selection, onSelect] = useSelectionInterface()
+  const resultIds = [...selection]
```

Comment:
:pencil2: would this work as well?
```suggestion
  const [resultIds, onSelect] = useSelectionInterface()
```

Reply:
Since the selection interface is a set rather than an array and the backend expects an array it has to be converted  with the `...` operator